plugins/outputs/postgresql/README.md

```diff
@@ -0,0 +1,54 @@
+# PostgreSQL Output Plugin
+
+This output plugin writes all metrics to PostgreSQL. 
+The plugin manages the schema automatically updating missing columns, and checking if existing ones are of the proper type. 
+
+**_WARNING_**: In order to enable automatic schema update, the connection to the database must
+be established with a user that has sufficient permissions. Either be a admin, or an owner of the 
+target schema.
+
+
+### Configuration:
+
+```toml
+# Send metrics to postgres
+[[outputs.postgresql]]
+    ## specify address via a url:
+    ##   postgres://[pqgotest[:password]]@localhost[/dbname]\
```

Comment:
FYI, the "postgres://" style connection strings no longer work (they did in the prev PR). The "simple string" approach seems to work.